spec/n1_loader_spec.rb

```diff
@@ -213,6 +213,12 @@ def perform(elements)
                                                                            ])
     end
 
+    it "supports falsey argument values" do
+      expect(object.with_default_argument(anything: 2)).to eq([object, [], 2])                      # default value
+      expect(object.with_default_argument(something: false, anything: 2)).to eq([object, false, 2]) # false
+      expect(object.with_default_argument(something: nil, anything: 2)).to eq([object, nil, 2])     # nil
+    end
```

Comment:
Can we also test that the caching works fine, please?

Reply:
:+1: